cdap-docs/examples-manual/source/examples/index.rst

```diff
@@ -45,6 +45,9 @@ In addition to the :ref:`Getting Started's <getting-started-index>`
   * - :doc:`File Sets<fileset>`
     - A variation of the *WordCount* example that operates on files. It demonstrates the usage of the FileSet
       dataset, including a service to upload and download files, and a MapReduce that operates over these files.
+  * - :doc:`Log Analysis <log-analysis>`
```

Comment:
This file (log-analysis) needs to be added to the table of contents, otherwise Sphinx can't find it. Right now, it is throwing this error:
```
checking consistency... /Users/john/Source/cdap_3.1/cdap-docs/examples-manual/source/examples/log-analysis.rst:: WARNING: document isn't included in any toctree
```
solution is to add it to the top of this file:
```
.. toctree::
   :maxdepth: 1
   :titlesonly:

   Hello World <hello-world>
   Count Random <count-random>
   File Sets <fileset>
   Log Analysis <log-analysis>
   Purchase <purchase>
   Spark K-Means <spark-k-means>
   Spark Page Rank <spark-page-rank>
   Sport Results <sport-results>
   Stream Conversion <stream-conversion>
   User Profiles <user-profiles>
   Web Analytics <web-analytics>
   Word Count <word-count>
```